evm/src/LinkToken.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Can you give some background on the case change of this file?

Reply:
To keep it consistent with the casing we use for our other contract artifacts since they're usually pascal case by convention